app/handlers/base.py

```diff
@@ -136,7 +139,8 @@ def Session(self, verify=True):
         before every commit.
 
         """
-        with VerifiedSession(self.current_user, verify) as session:
+        with VerifiedSession(self.current_user) as session:
+            session.add(self.current_user)
```

Comment:
Did you ever figure out if this is the right way?

Reply:
I'm not 100% sure. Let me check out the docs (I recall reading this but I need to make sure).

Reply:
Yeah, they say we should use `add()` https://docs.sqlalchemy.org/en/14/orm/session_basics.html#adding-new-or-existing-items